nntrainer/app_context.cpp

```diff
@@ -47,8 +51,104 @@
 #include <nnstreamer_layer.h>
 #endif
 
+/// add #ifdef across platform
+static std::string layerlib_suffix = "layer.so";
+static const std::string func_tag = "[AppContext] ";
+
+constexpr const char *DEFAULT_CONF_PATH = "/etc/nntrainer.ini";
+
+constexpr const char *getConfPath() {
+#ifdef NNTRAINER_CONF_PATH
+  return NNTRAINER_CONF_PATH;
+#endif
+  return DEFAULT_CONF_PATH;
+}
+
 namespace nntrainer {
 
+namespace {
```

Comment:
name of the namespace?

Reply:
This is intended, https://en.cppreference.com/w/cpp/language/namespace#Unnamed_namespaces